source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -450,6 +455,12 @@ ShaderGraphPtr ShaderGraph::create(const ShaderGraph* parent, const string& name
                 if (bindParamValue)
                 {
                     inputSocket->setValue(bindParamValue);
+
+                    input->setFlags(input->getFlags() | input->BIND_INPUT);
```

Comment:
Since there are already accessor methods for ShaderPort:setEmitted() and ShaderPort::isEmitted(), I would recommend adding corresponding methods for ShaderPort::setBindInput() and ShaderPort::isBindInput(), so that clients don't need to use bit-wise operators.